### ISSUE TITLE
NuxtPicture does not accept multiple formats

### ISSUE THREAD

**Issue by StijnCoolen** [nuxt/image#965]
When using `NuxtPicture` with ` format="avif,webp"`, the browser always seems to load a jpeg. However, when I use `NuxtImg` with the same properties an actual webp is loaded:

<img width="1165" alt="Scherm­afbeelding 2023-09-05 om 15 00 14" src="https://github.com/nuxt/image/assets/32257515/556761ae-1bd6-4729-b7a0-f8017a4834ae">

**Comment by RebeccaStevens** [comment]
I've got the same issue but I'm always getting png images. (My source images are png, so I imagine that's why png instead of jpg)

**Comment by StevenJPx2** [comment]
Yes, this is the problem that I'm facing as well. Bump for visibility.
